core/arch/arm/mm/core_mmu.c

```diff
@@ -909,6 +909,15 @@ static void assign_mem_granularity(struct tee_mmap_region *memory_map)
 	}
 }
 
+static unsigned int get_va_width(void)
+{
+#ifdef ARM64
+	return 64 - __builtin_ctzl(CFG_LPAE_ADDR_SPACE_SIZE);
+#else
+	return 32;
+#endif
```

Comment:
```suggestion
    if (IS_ENABLED(ARM64) {
        return 64 - __builtin_ctzl(CFG_LPAE_ADDR_SPACE_SIZE);
    } else {
        return 32;
    }
```
(somehow the suggested changes interface won't let me insert tabs…)

Reply:
preferably without braces
```c
static unsigned int get_va_width(void)
{
	if (IS_ENABLED(ARM64)
		return 64 - __builtin_ctzl(CFG_LPAE_ADDR_SPACE_SIZE);
	else
		return 32;
}